examples/nlp/language_modeling/conf/megatron_gpt_quantization.yaml

```diff
@@ -20,21 +20,22 @@ trainer:
   precision: bf16 # 16, 32, or bf16
   enable_checkpointing: false
 
+model:
+  tensor_model_parallel_size: 1
+  pipeline_model_parallel_size: 1
+  restore_from_path: llama2-7b-fp16.nemo # Nemo file path
+
 quantization:
-  quantize_bmm1: false
-  algorithm: fp8 # int8_sq, fp8, int8, int4_awq, null
+  decoder_type: ${export.decoder_type} # gptnext, gpt2, llama
+  algorithm: fp8 # null, int8, int8_sq, fp8, int4_awq, w4a8_awq, int4
```

Comment:
Nice to have `int8`, `w4a8_awq`, `int4` but bear in mind these have never been tested.

I would opt for removing them from description until someone tests this first

Reply:
Sure